src/darwin/Framework/CHIPTests/CHIPSetupPayloadParserTests.m

```diff
@@ -109,15 +109,15 @@ - (void)testOnboardingPayloadParser_QRCode_NoError
     XCTAssertEqual(payload.setUpPINCode.unsignedIntegerValue, 2048);
     XCTAssertEqual(payload.vendorID.unsignedIntegerValue, 12);
     XCTAssertEqual(payload.productID.unsignedIntegerValue, 1);
-    XCTAssertFalse(payload.requiresCustomFlow);
+    XCTAssertEqual(payload.commissioningFlow, kCommissioningFlowStandard);
```

Comment:
Why the payload change from custom to standard?

Reply:
I changed to the standard whenever it was `XCTAssertFalse` before.